CMakeLists.txt

```
@@ -248,6 +248,10 @@ find_package(PkgConfig QUIET)
 
 find_package(CURL ${CURL_MINIMUM} REQUIRED)
 
+if("${CURL_VERSION}" VERSION_EQUAL "8.9.1")
```

Just pointing out that this would suffice.

```suggestion
if(CURL_VERSION VERSION_EQUAL "8.9.1")
```